tasty-query/shared/src/test/scala/tastyquery/TypeSuite.scala

```diff
@@ -1256,4 +1256,26 @@ class TypeSuite extends UnrestrictedUnpicklingSuite {
     assert(clue(childToString.allOverriddenSymbols.toList) == List(superToString, objectToString))
     assert(clue(childToString.nextOverriddenSymbol) == Some(superToString))
   }
+
+  def companionClassFullCycle(path: DeclarationPath)(using Context, munit.Location): Unit = {
+    val Class: ClassSymbol = resolve(path).asClass
+    val Object: ClassSymbol = resolve(path.asObj).asClass
```

Comment:
`Class` and `Object` are well-known things in `java.lang`, so those names are a bit confusing. Consider using `cls` and `moduleClass` instead, which would follow the convention used in the codebase.